src/Traits/RelativeKeywordTrait.php

```diff
@@ -21,19 +21,33 @@ trait RelativeKeywordTrait
     protected static $relativePattern = '/this|next|last|tomorrow|yesterday|midnight|today|[+-]|first|last|ago/i';
 
     /**
-     * Determine if there is a relative keyword in the time string, this is to
-     * create dates relative to now for test instances. e.g.: next tuesday
+     * Determine if there is just a time in the time string
      *
      * @param string $time The time string to check.
      * @return bool true if there is a keyword, otherwise false
      */
-    public static function hasRelativeKeywords($time)
+    public static function hasRelativeTime($time)
```

Comment:
Does this new method need to be public? This method checks that the input is a time, would its name better match what it does if it were called `isTimeExpression()` or `isOnlyTime()`?

Reply:
Good point. I've renamed to `isTimeExpression()` and made private. I had made it public as that is what `hasRelativeKeywords()` was.